Objects/Objects/Geometry/Point.cs

```diff
@@ -150,50 +150,36 @@ public void Deconstruct(out double x, out double y, out double z)
     z = this.z;
   }
 
-  public static Point operator +(Point point1, Point point2)
-  {
-    return new Point(point1.x + point2.x, point1.y + point2.y, point1.z + point2.z, point1.units);
-  }
+  public static Point operator +(Point point1, Point point2) =>
+    new(point1.x + point2.x, point1.y + point2.y, point1.z + point2.z, point1.units);
 
-  public static Point operator -(Point point1, Point point2)
-  {
-    return new Point(point1.x - point2.x, point1.y - point2.y, point1.z - point2.z, point1.units);
-  }
+  public static Point operator -(Point point1, Point point2) =>
+    new(point1.x - point2.x, point1.y - point2.y, point1.z - point2.z, point1.units);
 
-  public static Point operator *(Point point1, Point point2)
-  {
-    return new Point(point1.x * point2.x, point1.y * point2.y, point1.z * point2.z, point1.units);
-  }
+  public static Point operator *(Point point1, Point point2) =>
+    new(point1.x * point2.x, point1.y * point2.y, point1.z * point2.z, point1.units);
 
-  public static Point operator *(Point point, double val)
-  {
-    return new Point(point.x * val, point.y * val, point.z * val, point.units);
-  }
+  public static Point operator *(Point point, double val) =>
+    new(point.x * val, point.y * val, point.z * val, point.units);
 
-  public static Point operator /(Point point, double val)
-  {
-    return new Point(point.x / val, point.y / val, point.z / val, point.units);
-  }
+  public static Point operator /(Point point, double val) =>
+    new(point.x / val, point.y / val, point.z / val, point.units);
 
   public static bool operator ==(Point? point1, Point? point2)
   {
     if (point1 is null && point2 is null)
     {
       return true;
     }
-
-    if (point1 is null ^ point2 is null)
+    else if (point1 is null ^ point2 is null)
```

Comment:
Could we change this operator to be:

I don't know why we'd want to use the binary XOR operator... Since the previous case covers `&&`
And doing a `||` also makes the `?` in the property comparison unnecessary.

```cs
  public static bool operator ==(Point? point1, Point? point2)
  {
    if (point1 is null && point2 is null)
    {
      return true;
    }
    else if (point1 is null || point2 is null)
    {
      return false;
    }

    return point1.units == point2.units && point1.x == point2.x && point1.y == point2.y && point1.z == point2.z;
  }
  ```